### ISSUE TITLE
Curious page breaks in tables

### ISSUE THREAD

**Issue by franksauer** [asciidoctor/asciidoctor-pdf#871]
I'm on the way migrating some Latex projects to AsciiDoctor and have some issues.
One of my documents is a questionaire.

My tables for a questionaire are defined as

|===
2.5+|*Q-02* 18+|What is the correct answer?
| [A] 17+|_Option A is the correct answer_
| [B] 17+|_Option B is the correct answer_
| [C] 17+|_Option C is the correct answer_
| [D] 17+|_Option D is the correct answer_
|===

For some reasons the [cols=10,5,90] didn't work together with column-spans, so I found the solution above which works fine for me.

When I place several question-tables in the document, I got the result like the attached document (minimal example)
(First column on one page, rest of the table on the next page; in some cases the text of the question disappeared)
Is there a trick? I already tried [options=unbreakable], which did't change the behavior. The manual mentioned, that it only would work with DocBook.
Should I change my workflow to asciidoc -> docbook -> pdf?

Regards Frank

[test.pdf](https://github.com/asciidoctor/asciidoctor-pdf/files/1507688/test.pdf)

**Comment by mojavelinux** [comment]
This is a known bug in Prawn table. Asciidoctor PDF doesn't actually handle laying out the table. That is up to the Prawn table library. This is a duplicate of https://github.com/asciidoctor/asciidoctor-pdf/issues/403 and has been reported upstream.

**Comment by mojavelinux** [comment]
Asciidoctor PDF does not support unbreakable tables. One reason is that it's far too expensive. The second reason is that the table layout is handled by Prawn table and it really needs to be added there first before we can use it.

**Comment by graphitefriction** [comment]
Related to #2022

**Comment by graphitefriction** [comment]
In the upcoming release of Asciidoctor PDF 2.0, you will be able to prevent an orphaned table caption by assigning the option `breakable` to a table.

```
.Optional table caption/title
[%breakable]
|===
|Content |Content

|Content |Content
|===
```

Note that the table will still break if it flows from the current page to the next page. The `breakable` option only prevents the table caption (and ID, if one is assigned) from being orphaned. (We decided it was best if people choose to opt-in to this behavior due to the potential performance costs in some cases.)

Also, you'll be able to assign the `unbreakable` option to a table if you don't want it to break and flow to the next page when it doesn't fit on the current page.

```
.Optional table caption/title
[%unbreakable%header,cols="2*"]
|===
|Content
|Content

|Content
|Content
|===
```
